src/commons/js/index.js

```diff
@@ -79,6 +81,24 @@
         });
     }
 
+    function fetchClusterData() {
+        var initInjector = angular.injector(["ng"]);
+        
+        var $http = initInjector.get("$http");
+
+        return $http.get("/api/GetClusterList.json").then(function(response) {
```

Comment:
later , once we will have api . than you suppose to add that api here . but what about "base_url" which is coming from config.json . you have to use that base_url and some uri , in order to call the cluster-list API .
have you tried this?